runtime/vam/expr/function/gendatepartfuncs.go

```diff
@@ -32,16 +32,20 @@ case *vector.View:
 	out := make([]int64, len(index))
 	for i, idx := range index {
 		v := inner.Values[idx]
-		out[i] = %s
+		out[i] = %[1]s
```

Comment:
See https://pkg.go.dev/fmt#hdr-Explicit_argument_indexes for details on this %[1]s business.